packages/node/src/meta/meta.service.ts

```diff
@@ -47,6 +53,17 @@ export class MetaService {
     };
   }
 
+  @Interval(UPDATE_HEIGHT_INTERVAL)
+  async checkTargetHeight() {
+    await Promise.all([
+      this.storeService.setMetadata(
+        'lastProcessedHeight',
+        this.lastProcessedHeight,
+      ),
+      this.storeService.setMetadata('targetHeight', this.targetHeight),
```

Comment:
@ianhe8x I added targetHeight here, otherwise stored in db every 5 seconds. Is this interval fine?

Reply:
lgtm